### ISSUE TITLE
Add support for GUID-less URLs

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#16]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To avoid identifiers in URLs of repetitive pages.

### Why

IDs can be avoided with the alternative URLs feature. However, they require the editor to state the whole relative path in the alternative URL. For each new page. This is tedious and cumbersome.

Recently, I developed a similar [mechanism that requires to specify just the final slug](https://github.com/Kentico/cms-ems-samples/tree/guid-less-urls) (segment) in the whole relative path. This mechanism also has a collision-prevention logic that pings the live site to ultimately decide whether the final URL can be used or not.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
